tests/userController.test.js

```diff
@@ -14,6 +14,7 @@ describe("user controller", () => {
     res = {
       status: jest.fn().mockReturnThis(),
       json: jest.fn().mockReturnThis(),
+      send: jest.fn(),
```

Comment:
I'll approve but curious on why you added this. I dont see any test using `res.send` and instead they are all using `res.json`.

Reply:
There were tests that had { send: jest.fn() } inside of their `res`. When I removed the `res` from those tests the tests would stop working. In order to avoid this I left the `send: jest.fn()` inside  of the `beforeEach`

Reply:
Oh, I see what happened now. The other branchs had `send: jest.fn()` inside of them while this one had `json: jest.fn()`, I assumed that this also had `send: jest.fn()` and did the same thing that I had done to the other branch, that being deleting the `res` from each tests and adding `send: jest.fn()` in the `beforeEach`